### PR TITLE
Add support to AWS Prowler V3 json format

### PR DESCRIPTION
**Pull request by anderson-slompo**
**Description**
Add support to import aws prowler v3 json scan file referent to this [issue](https://github.com/DefectDojo/django-DefectDojo/issues/7348)

### CODE REVIEW

File: dojo/tools/aws_prowler_v3/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'Anderson dos Anjos Slompo - https://github.com/anderson-slompo'
```
Comment (damiencarol) [inline dojo/tools/aws_prowler_v3/__init__.py:1]:
```suggestion
```

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: dojo/tools/aws_prowler_v3/parser.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            provider = deserialized.get('Provider')
+            compliance = str(deserialized.get('Compliance'))
+            result_extended = deserialized.get('StatusExtended')
+            general_description = deserialized.get('Description')
+            asff_compliance_type = " / ".join(deserialized.get('CheckType'))
+            severity = deserialized.get('Severity').capitalize()
+            aws_service_name = deserialized.get('ServiceName')
+            impact = deserialized.get('Risk')
+            mitigation = deserialized.get('Remediation').get('Recommendation').get("Text")
+            mitigation = str(mitigation) + "\n" + str(deserialized.get('Remediation').get('Code'))
```
Comment (Maffooch) [inline dojo/tools/aws_prowler_v3/parser.py:49]: Using chained `.get()` can end poorly if the expected field is not found. For example if `Remediation` is not present, the line becomes `str(None.get("Code"))` and would throw an exception. Please update all other references of `.get()` chains
```suggestion
            mitigation = str(mitigation) + "\n" + str(deserialized.get('Remediation', {}).get('Code', ''))
```

File: dojo/tools/aws_prowler_v3/parser.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    find.description += description + "\n\n"
+                find.nb_occurences += 1
+            else:
+                find = Finding(
+                    title=textwrap.shorten(result_extended, 150),
+                    cwe=1032,  # Security Configuration Weaknesses, would like to fine tune
+                    test=test,
+                    description=description,
+                    component_name=component_name,
+                    unique_id_from_tool=unique_id_from_tool,
```
Comment (Maffooch) [inline dojo/tools/aws_prowler_v3/parser.py:86]: Use this to update the deduplication settings
https://documentation.defectdojo.com/contributing/how-to-write-a-parser/#deduplication-algorithm

**Review by Maffooch — CHANGES_REQUESTED** [review]

**Comment by anderson-slompo** [comment]
@Maffooch can you validate deduplication settings ?

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

File: dojo/settings/settings.dist.py [hunk 4]
```diff
@@ -1173,6 +1173,7 @@ def saml2_attrib_map_format(dict):
     'Anchore Enterprise Policy Check': ['title', 'severity', 'component_name', 'file_path'],
     'Anchore Grype': ['title', 'severity', 'component_name', 'component_version'],
     'Aqua Scan': ['severity', 'vulnerability_ids', 'component_name', 'component_version'],
+    'AWS Prowler V3': ['severity', 'component_name', 'unique_id_from_tool'],
```
Comment (Maffooch) [inline dojo/settings/settings.dist.py:1176]: Adding `unique_id_from_tool` to the hash code fields will remove the possibility of having deduplication across tools. Please either remove it or replace it with another field
```suggestion
    'AWS Prowler V3': ['severity', 'component_name'],
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

**Review by Maffooch — DISMISSED** [review]

**Review by Maffooch — APPROVED** [review]
Approved for real this time!

**Review by devGregA — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Comment by mtesauro** [comment]
@damiencarol Are you good with this?

**Comment by mtesauro** [comment]
@damiencarol Are you OK with this?

**Comment by finventario** [comment]
Thanks a lot guys!

**Pull request merged**